news/editions/edition3.md

```diff
@@ -0,0 +1,67 @@
+
+## [How to Build a Centos RPM Via a Spinnaker "Bake"](https://blog.opsmx.com/how-to-build-bake-centos-rpm-using-spinnaker/)
+While Spinnaker doesn't offer a CentOS baking option by default, this tech blog demonstrates the use of package management and S3 to bake a build as a CentOS image, and deploying an instance of the baked image on AWS cloud. With the RPM built, configuring and executing the bake in Spinnaker requires just a few steps.
+
+## ['Just Me and Opensource' Tutorials Available on Youtube](https://www.youtube.com/watch?v=9EUyMjR6jSc&t=2s)
+Fans of the __[Just me and Opensource](https://www.youtube.com/user/wenkatn)__ online tutorials got a nice dose of CD when the popular Youtuber published a new segment of his Kubernetes series focused on deploying Spinnaker to Kubernetes. Brilliant to learn hands-on how these open source software tools and technologies are better together.
+
+## [Netflix's Interactive Slack Notifications for ChatOps with Spinnaker](https://blog.spinnaker.io/interacting-with-spinnaker-via-slack-at-netflix-9ab262e8218d)
+2-way interactive Slack notifications allow Netflix engineers deploying with Spinnaker's new Managed Delivery feature to approve a Spinnaker pipeline manual judgement directly from Slack. Learn how Echo fields communications between notifying services and Slack via interactive notification callbacks.
+
+## [Operating Spinnaker at Scale & the Ops SIG's Mission](https://www.armory.io/blog/scaling-spinnaker-at-salesforce-the-life-of-a-cloud-ops-architect/)
+Discover the mission of the new Operations SIG in this interview with the co-chair, Edgar Magana of Salesforce. This SIG will address several goals and challenges, including reference architecture. Edgar explains these through the lens of Salesforce's Spinnaker implementation, which targets both Kubernetes and EC2 deployments.  
+
+## [Adopting the V2 K8S Provider As V1 Support Sunsets](https://blog.spinnaker.io/farewell-to-the-kubernetes-v1-provider-79d93861c6e4)
+Spinnaker 1.21, expected near the end of June, will be the final release that includes the Kubernetes V1 provider, as community best practices optimize for V2. Read more about generating V2-compliant, version-controlled Kubernetes manifests, and additional steps to migrate pipelines.
+
+## [PaCRD Deep Dive: Try Out Spinnaker Pipelines as CRDs](https://www.youtube.com/watch?v=HZtRCyGA7yM)
```

Comment:
This is Armory proprietary I thought. It shouldn't be in the OSS highlights if it is.

Reply:
My thought on this was, this is an experimental feature that has not been monetized. There are other parallel efforts to experiment w Spinnaker pipelines as CRDs but no content out there around it that I've found other than this. The CDF's perspective seems to be, it's OK to talk about proprietary features built onto OSS tools as it's all part of the ecosystem. Trying to be judicious in my bias towards Armory, but also feel it's important to highlight that Spinnaker does have an evolving story beyond the GUI and that it's in active development (as that is one argument you hear a lot as a count for tekton/jenkins X vs Spinnaker). Happy to hear more thoughts!

Reply:
@plumpy, for a ruling.

@dnilasor if the fact that it hasn't been optimized yet is relevant to the question of appropriateness, consider that it might be monitized later and this content will still be here.

Reply:
I don't have a problem with that personally (thinking with my OSS user/contrib brain and not my armory brain) but am happy to be overruled. To me getting visibility into our PaC story is worth it for competitive purposes regardless.

Reply:
Yeah, I appreciate you calling this out, but I'm inclined to say it's fine...

Reply:
👍